private_repo/ansible/inventory/dev/Core/common.yml

```diff
@@ -236,10 +236,11 @@ cloud_artifact_storage_project: "{{ cloud_public_storage_project }}"
 # cloud_storage_redisbackup_bucketname: ""
 
 # Building block vars
-cloud_storage_base_url: "{{ cloud_storage_url }}"
-cloudstorage_base_path: "{{ cloud_storage_url }}"
-valid_cloudstorage_base_urls: '["{{ cloud_storage_url }}"]'
+cloud_storage_base_url: ""
+cloudstorage_base_path: ""
+valid_cloudstorage_base_urls: '["https://namespace.compat.objectstorage.region.oraclecloud.com",""]'
```

Comment:
@ddevadat should we initialise this with other csp storage account ? Can this be replaced with {{ cloud_storage_url }}

Reply:
corrected this

Reply:
Change is not reflecting in the PR yet